policyengine_us/parameters/gov/states/mo/tax/income/credits/property_tax/property_tax_limit.yaml

```
@@ -10,5 +10,5 @@ metadata:
       href: https://dor.mo.gov/forms/MO-PTS_2023.pdf#page=2
     - title: 2021 MO-PTS Property Tax Credit Schedule, Line 11
       href: https://dor.mo.gov/forms/MO-PTS_2021.pdf#page=2
-    - title: MO Rev 135.025
+    - title: Revisor of Missouri Title X TAXATION AND REVENUE Chapter 135.025
```

```suggestion
    - title: Missouri Rev 135.025 (2)
```
For example, the value 1100 appears in section 2 of the link. Therefore we add (2) at the end of the title. If the value appears in section 1 part 2, then it is Missouri Rev 135.025 (1)(2)

---

Got it!